ProjectVoyage/Cell/MessageCell.h

```diff
@@ -19,18 +19,7 @@ for incoming or outgoing messages
 @interface MessageCell : UITableViewCell
 
 @property (nonatomic, strong) Message *message;
-
-@property (weak, nonatomic) id <ContainerProtocol> delegate;
-
-// Container
-@property (strong, nonatomic) UILabel *messageType;
-@property (strong, nonatomic) UIButton *deleteButton;
-@property (strong, nonatomic) UIButton *editButton;
-@property (strong, nonatomic) UIButton *changeSenderButton;
-@property (strong, nonatomic) UITextView *textView;
-@property (strong, nonatomic) UIImageView *bubbleImage;
```

Comment:
These should be moved to the implementation file (the public API should be as clean as possible)